data/categories.txt

```diff
@@ -3864,7 +3866,7 @@ nb:Post
 pl:4Poczta
 pt:3Correios
 pt-BR:3Correios
-ro:Poştă
+ro:Poştă|oficiu poștal
```

Comment:
The `@post` search group is used for postboxes and parcel lockers also, so it will be inappropriate to display parcel lockers amongst search results if someone used an "oficiu poștal" alias.
Please add this alias directly to `amenity-post_office` type instead.